src/app/credExplorer/RepositorySelect.js

```diff
@@ -95,17 +93,17 @@ export async function loadStatus(localStore: LocalStore): Promise<Status> {
       return {type: "FAILURE"};
     }
     const json = await response.json();
-    let availableRepos = Object.keys(json).map(repoStringToRepo);
-    availableRepos = sortBy(availableRepos, (r) => r.owner, (r) => r.name);
+    const availableRepos = fromJSON(json);
```

Comment:
(Here is another place where it’s nice that we wrote `fromJSON`/`toJSON`
instead of forcing clients to work with strings.)